foubus_test.py

```diff
@@ -27,7 +25,7 @@ def setUp(self):
         os.symlink(os.path.relpath("style.css", d), f"{d}/style.css")
         os.chdir(d)
         foubus.download()
-        foubus.build_stop_timetable(datetime.date(2025, 1, 18))
+        foubus.build_stop_timetable(datetime.date(2025, 12, 24))
```

Comment:
If I understand correctly, these test changes did not fix the test. Then please revert the changes.